JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt

```diff
@@ -168,13 +167,12 @@ private fun CodeBlockParagraph(
         color = codeBlockBackground,
         shape = RoundedCornerShape(4.dp)
     ) {
-        Padding(16.dp) {
-            Text(
-                text = text,
-                style = textStyle,
-                paragraphStyle = paragraphStyle
-            )
-        }
+        Text(
+            modifier = Spacing(top = 16.dp),
```

Comment:
Are you sure about spacing top here? The padding was all around the text

Reply:
Good catch